.eslintrc.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     'eslint:recommended',
     'plugin:@typescript-eslint/recommended',
     'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:import/recommended',
+    // 'plugin:import/recommended', // This is causing a problem with importing types from aws-lambda package
```

Comment:
I haven't spent much time looking into what the problem is, but VScode says it cannot resolve the module even though it is installed.

Do we need this setting?

Reply:
Take a look at the AWS handler in `airnode-deployer`. If you prefix the type with the namespace you apparently don't need to import anything.

Reply:
Interesting! It turns out we don't need these types at all though.